src/main/java/ai/deepcode/javaclient/core/AnalysisDataBase.java

```diff
@@ -594,7 +590,7 @@ private GetAnalysisResponse doGetAnalysis(
       }
 
       counter++;
-    } while (!response.getStatus().equals("DONE")
+    } while (!response.getStatus().equals("COMPLETE")
```

Comment:
lets ignore casing when doing string equality, in case API decides to send response in lower case ;-)